docs/content/preview/troubleshoot/cluster/failed_tablets.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Recover YB-TServer from crash loop
+linkTitle: Recover YB-TServer from crash loop
+description: Recover YB-TServer from crash loop
+menu:
+  preview:
+    parent: troubleshoot-cluster
+    weight: 866
+isTocNested: true
+showAsideToc: true
+---
+
+When a YB-TServer process or node has failed, YugabyteDB will automatically trigger a remote bootstrap for
+ most types of tablet data corruption or failures. However, there are still a number of cases where this automatic mechanism might not be able to help.
+
+One major such case is anything that leads to a code path that we've encoded as a crash (eg: `CHECK` or `FATAL`) or is an unknown unknown 
```

Comment:
"One major such case is anything that leads to a code path that we've encoded as a crash (eg: `CHECK` or `FATAL`) or is an unknown unknown 
that causes a crash (eg: code bugs leading to `SIGSEGV`). In all of these cases, it's likely the root cause would repeat itself, when the process is brought back up! 
Moreover, when using YugabyteDB Anywhere, crashed processes will be automatically restarted, to ensure minimum downtime! "   >

One major cause of such case would be a condition that leads to a code path encoded as a crash (for example, `CHECK` or `FATAL`) or the causes of a crash is unknown (for example, code errors that lead to `SIGSEGV`). In all of these cases, the root cause would likely repeat itself when the process is restarted. 
Moreover, when using YugabyteDB Anywhere, crashed processes are automatically restarted to ensure minimum downtime.